api/client/build.go

```diff
@@ -211,15 +211,12 @@ func (cli *DockerCli) CmdBuild(args ...string) error {
 		}
 	}
 
-	var shmSize int64 = 67108864 // initial SHM size is 64MB
+	shmSize := runconfig.DefaultSHMSize
```

Comment:
I also think I can move the const to daemon or container as well since it isn't used in runconfig anymore